string sender
nav_msgs/GridCells cells
